server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -1211,9 +1212,12 @@ public int getNumberOfReplicas() {
 
     /**
      * Returns true if segment replication is enabled on the index.
+     *
+     * Every shard on a remote node would also have SegRep enabled even without
+     * proper index setting during the migration.
      */
     public boolean isSegRepEnabled() {
```

Comment:
lets change the name of this method since we have a segrep local method already.